packages/app/src/app/pages/Sandbox/Editor/Header/PrivacyTooltip/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { useOvermind } from 'app/overmind';
+import { ThemeProvider } from 'styled-components';
+import Tooltip from '@codesandbox/common/lib/components/Tooltip';
+import theme from '@codesandbox/common/lib/design-language/theme';
+import { Container, Text, Link, Select } from './elements';
+import * as Icons from './icons';
+
+export const PrivacyTooltip = () => {
+  const {
+    actions: {
+      workspace: { sandboxPrivacyChanged },
+    },
+    state: {
+      user: { subscription },
+      editor: {
+        currentSandbox: { privacy },
+      },
+    },
+  } = useOvermind();
+
+  const config = {
+    0: {
+      description: 'Everyone can see this Sandbox',
+      icon: <Icons.Public />,
+    },
+    1: {
+      description:
+        'Only people with a private link are able to see this Sandbox',
+      icon: <Icons.Unlisted />,
+    },
+    2: {
+      description: 'Only you can see this Sandbox.',
+      icon: <Icons.Private />,
+    },
+  };
+
+  const onChange = event => {
+    const value = event.target.value;
+    sandboxPrivacyChanged(Number(value) as 0 | 1 | 2);
+  };
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Container>
+        <Tooltip
+          delay={0}
+          interactive
+          content={
+            <>
+              <Text size="3" marginBottom={4}>
+                {subscription ? (
+                  'Adjust privacy settings.'
+                ) : (
+                  <>
+                    You can change privacy of a sandbox as a Pro.
+                    <br />
+                    <Link href="/pricing">Upgrade to Pro</Link>
+                  </>
+                )}
+              </Text>
+
+              <Select
+                marginBottom={2}
+                value={privacy}
```

Comment:
@christianalfoni Could use some help here. 

```
Property 'value' does not exist on type 'IntrinsicAttributes'
```

I'm not using value inside the styled component, but we need to type this inside the element?

Reply:
I think you do because the value tag does not exist in the select tag, its a react invention to keep us crying

Reply:
ohhh, the semantic version is have a `<option selected>`

should we use that instead?

Reply:
if we have that react complains so it's all about who you want to hear complain ahah

Reply:
oh lol, didn't know that. okay then i'll prefer typescript complaining

Reply:
Yeah, just add the value to the styled-component

Reply:
Ah, `as` support in TypeScript strikes again! I've pushed a commit which forces the component to be seen as a `select`, but this is really not ideal. @christianalfoni what do you think?